fastpath/fastpath/core.py

```diff
@@ -1353,32 +1353,48 @@ def score_signal(msm: dict) -> dict:
     try:
         # https://github.com/ooni/probe/issues/2344
         tv = g_or(msm, "test_version", "0.0.0")
-        if parse_version(tv) <= parse_version("0.2.3"):
+
+        msmt_start_time = msm.get("measurement_start_time")
+        if msmt_start_time is None:
+            scores["accuracy"] = 0.0
+            return scores
+        start_time = datetime.strptime(msmt_start_time, "%Y-%m-%d %H:%M:%S")
+
+        if parse_version(tv) <= parse_version("0.2.4") and parse_version(
+            tv
+        ) > parse_version("0.2.3"):
```

Comment:
```suggestion
        if parse_version(tv) == parse_version("0.2.4"):
```

IIUC, we are saying that we're not sure about version 0.2.4 and we can just safely drop it, right? Then, in such a case, I suggest it's better to just check for equality here.